setup.py

```diff
@@ -35,10 +41,6 @@ def get_install_requires():
             continue
         # add line to requirements
         requirements.append(line.replace('\n', ''))
-    # add py2-ipaddress if python2
```

Comment:
I was wrong again, that bit of code is necessary for non-wheel packages (old fashion tar.gz).